data/json/itemgroups/tools.json

```diff
@@ -263,6 +263,11 @@
       [ "whistle", 20 ],
       [ "hand_crank_charger", 15 ],
       [ "flint_steel", 5 ],
+      [ "hobo_stove", 3 ],
```

Comment:
This group seems oriented around a modern, well-outfitted survivalist, the hobo stove, tinder, and wooden canteen seem out of place.